ganttproject/src/main/java/net/sourceforge/ganttproject/task/TaskManagerImpl.java

```diff
@@ -149,10 +150,17 @@ public void removeAllTasks(Iterable<Task> tasks) {
 
   private final CustomColumnsManager myCustomColumnsManager;
 
+  private final TaskUpdateBuilder.Factory myTaskUpdateBuilderFactory;
   private Boolean isZeroMilestones = true;
 
   public TaskManagerImpl(TaskContainmentHierarchyFacade.Factory containmentFacadeFactory, TaskManagerConfig config) {
+    this(containmentFacadeFactory, config, null);
+  }
+
+  public TaskManagerImpl(TaskContainmentHierarchyFacade.Factory containmentFacadeFactory, TaskManagerConfig config,
```

Comment:
I thought it should be better to add it here rather than setting it up in the `TaskManagerConfig` because there're multiple config implementations and only one `TaskManagerImpl`.